css-values-4/Overview.bs

```diff
@@ -1063,33 +1063,69 @@ URL Processing Model</h4>
 
 		4. Let |base| be |sheet|'s <a spec=cssom>stylesheet base URL</a>. [[CSSOM]]
 
-		5. Let |referrer| be |documentBase|.
+		5. If |base| is null, set |base| to |documentBase|.
 
-		6. If |base| is null, set |base| to |documentBase|.
+		6. Let |url| be |urlValue|.
 
-		7. Let |parsedUrl| be the result of the [=URL parser=] steps with |url| and |base|.
-			If the algorithm returns an error, return.
+		7. If |urlValue| is a <<url>>, set |url| to |urlValue|'s [/=url].
 
-		8. If |corsMode| is "cors", set |referrer| to |sheet|'s
-			<a spec=cssom>location</a>. [[CSSOM]]
+		8. Let |parsedUrl| be the result of the [=URL parser=] steps with |url| and |base|.
+			If the algorithm returns an error, return.
 
 		9. Let |req| be a new [=/request=] whose [=request/url=] is |parsedUrl|, whose
 			[=request/destination=] is |destination|, [=request/mode=] is |corsMode|,
-			[=request/origin=] is |environmentSettings|'s [=environment settings object/origin=],
-			[=request/credentials mode=] is "same-origin", [=request/use-url-credentials flag=] is
-			set, [=request/client=] is |environmentSettings|, and whose [=request/referrer=] is
-			|referrer|.
+			[=request/origin=] is |environmentSettings|'s
+			[=environment settings object/origin=], [=request/credentials mode=] is
+			"same-origin", [=request/use-url-credentials flag=] is set, [=request/client=] is
+			|environmentSettings|, and whose [=request/referrer=] is |documentBase|.
 
-		10. If |sheet|'s <a spec=cssom>origin-clean flag</a> is set, set |req|'s
+		10. [=Apply request modifiers from URL value=] given |req| and |urlValue|.
+
+		11. If |req|'s [=request/mode] is "cors", set |req|'s [=request/referrer=] to |sheet|'s
+			<a spec=cssom>location</a>. [[CSSOM]]
+
+		12. If |sheet|'s <a spec=cssom>origin-clean flag</a> is set, set |req|'s
 			[=request/initiator type=] to "css". [[CSSOM]]
 
-		11. [=/Fetch|fetching=] |req|, with [=fetch/processresponseconsumebody=] set to
+		13. [=/Fetch|fetching=] |req|, with [=fetch/processresponseconsumebody=] set to
 			|processResponse|.
 	</div>
 
-	Note: Resources loaded through CSS style sheets
-	are cached and cleared the same as any other resources
-	linked from the document.
+<h4 id='request-url-modifiers'>
+Request URL Modifiers</h4>
+
+	The <dfn><<request-url-modifier>></dfn>s represent <<url-modifier>>s that affect the resource's
+	[=/request=]. Each <<request-url-modifier>> has a <dfn>request modifier steps</dfn> which accepts
+	a [=/requests=].
```

Comment:
```suggestion
	a [=/request=].
```

Also gonna stop rewrapping in suggestions, just know that all of these paragraphs should be using semantic breaks. ^_^